internal/pkg/service/buffer/store/receiver_test.go

```diff
@@ -4,15 +4,110 @@ import (
 	"context"
 	"sort"
 	"testing"
+	"time"
 
+	"github.com/keboola/go-client/pkg/storageapi"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/filestate"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/slicestate"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
 )
 
+func TestStore_CreateReceiverOp(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
```

Comment:
Pridal som este unit test na vytvorenia receiver + 2 exports.
Je na to E2E test, ale lepsie sa to debuguje ako unit test.